prime-router/metadata/fhir_mapping/hl7/datatypes/ID/CodeableConcept.yml

```diff
@@ -15,6 +15,6 @@ coding:
           expressionsMap:
             url:
               type: SYSTEM_URL
-              value: hl7-use
-            valueString:
-              valueOf: "id-code"
\ No newline at end of file
+              value: codeable-concept-id
+            valueBoolean:
```

Comment:
This was just operating as marker in the array of `coding` and didn't really match to a field, component or segment so I gave it a dedicated URL